src/modules/Rating/RatingIcon.js

```diff
@@ -52,7 +70,16 @@ export default class RatingIcon extends Component {
       useKeyOnly(selected, 'selected'),
       'icon'
     )
+    const rest = _.omit(this.props, _.keys(RatingIcon.propTypes))
 
-    return <i className={classes} onClick={this.handleClick} onMouseEnter={this.handleMouseEnter} />
+    return (
+      <i role='radio' tabIndex={0}
+        {...rest}
+        className={classes}
+        onKeyPress={this.handleKeyPress}
```

Comment:
Ah, yes, I debated this a bunch originally. I was under the impression key press only fired after a key down followed by a key up which is why i chose it. But I could be wrong, which would make key up a good alternative. Unfortunately because of the holiday I'm not certain when I'll be able to make the update. Feel free to change it if you have time, otherwise I'll pick it back up next week at the latest